server/sonar-web/src/main/js/components/issue/Issue.js

```diff
@@ -108,9 +108,9 @@ export default class Issue extends React.PureComponent {
       this.togglePopup('edit-tags');
       return false;
     });
-    key('space', 'issues', () => {
+    key('space', 'issues', (event /*: Event*/) => {
       if (this.props.onCheck) {
-        this.props.onCheck(this.props.issue.key);
+        this.props.onCheck(this.props.issue.key, event);
```

Comment:
I wonder if it'd better to not pass the whole event, but just the `shiftKey` flag. WDYT?

Reply:
I did it that way because in case we want to use more keys like Ctrl or Alt to improve the select behavior later with new functionalities.